docs/root/intro/arch_overview/other_protocols/redis.rst

```diff
@@ -90,6 +90,11 @@ Every Redis cluster has its own extra statistics tree rooted at *cluster.<name>.
 
   max_upstream_unknown_connections_reached, Counter, Total number of times that an upstream connection to an unknown host is not created after redirection having reached the connection pool's max_upstream_unknown_connections limit
   upstream_cx_drained, Counter, Total number of upstream connections drained of active requests before being closed
+  upstream_commands.upstream_rq_time, Histogram, Histogram of upstream request times for all types of requests
```

Comment:
Can you break this into a second table and make it more clear that these only show up if a config setting is toggled (and also ref link to that setting)?